dataactbroker/fileRoutes.py

```diff
@@ -130,6 +130,70 @@ def get_protected_files():
         file_manager = FileHandler(request, is_local=is_local, server_path=server_path)
         return file_manager.get_protected_files()
 
+    @app.route("/v1/check_current_page/", methods=["GET"])
+    @convert_to_submission_id
+    def check_submission(submission_id):
+        sess = GlobalDB.db().session
+        # /v1/reviewData/
+        reviewdata = sess.query(Job).filter(Job.submission_id == submission_id,
```

Comment:
we should be using underscoring as the naming convention for variables. So `reviewdata` should be `review_data`. It is more readable as opposed to being all one word.